src/aiperf/common/config/loadgen_config.py

```
@@ -18,6 +18,87 @@ class LoadGeneratorConfig(BaseConfig):
+            parts = [part.strip() for part in v.split(",")]
+
+            # Single value without comma - return as int
+            if len(parts) == 1:
+                try:
+                    return int(parts[0])
+                except ValueError as err:
+                    raise ValueError(
+                        f"Invalid concurrency value: '{parts[0]}'. "
+                        f"Must be a positive integer (>= 1). "
+                        f"Examples: --concurrency 10, --concurrency 20"
+                    ) from err
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)